public/js/simple-mode.js

```diff
@@ -147,9 +147,10 @@ function simulate(
     const numberOfMetrics = metrics.length
     for (let i = 0; i < numberOfMetrics; i++) {
         const metric = metrics[i]
-        const percentage = getBudgetPercentageForMetricIdFromDom(i + 1)
+        const isPercentage = true
```

Comment:
Is this because the simple mode only supports percentage?

Reply:
Yes, that's correct. Because I changed the getScalingFactorMetric to calculate the scaling factor differently depending on whether percentage or value is selected.